moneyed/classes.py

```diff
@@ -1,28 +1,40 @@
 # -*- coding: utf-8 -*-
+
 from __future__ import division
 from __future__ import unicode_literals
 
 from decimal import Decimal, ROUND_DOWN
+import sys
+import warnings
 
 # Default, non-existent, currency
 DEFAULT_CURRENCY_CODE = 'XYZ'
 
+PYTHON2 = sys.version_info[0] == 2
+
 
 class Currency(object):
     """
     A Currency represents a form of money issued by governments, and
     used in one or more states/countries.  A Currency instance
     encapsulates the related data of: the ISO currency/numeric code, a
-    canonical name, countries the currency is used in, and an exchange
-    rate - the last remains unimplemented however.
+    canonical name, and countries the currency is used in.
     """
 
-    def __init__(self, code='', numeric='999', name='', countries=[]):
+    def __init__(self, code='', numeric='999', name='', countries=None):
+        if countries is None:
+            countries = []
         self.code = code
         self.countries = countries
         self.name = name
         self.numeric = numeric
 
+    def __eq__(self, other):
```

Comment:
We already have an `__eq__` method. We should drop it and use this, upstream one.